### ISSUE TITLE
improvement

### ISSUE THREAD

**Issue by kd-dinesh** [tjsudarsan/trackexpense#1]
Need to provide option for different cards, another filter by cards/banks/cash, reccuring expenses.
Also search is not working on note section

**Comment by tjsudarsan** [comment]
> Need to provide option for different cards, another filter by cards/banks/cash, reccuring expenses.
> Also search is not working on note section

Yeah @kd-dinesh I am planning to completely rewind the app with lot exiciting features! Your suggestions will be definitely on that. Thanks Man!
